x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/manifest.json

```diff
@@ -1,29 +1,29 @@
 {
   "id": "apm_transaction",
   "title": "APM",
-  "description": "Detect anomalies in high mean of transaction duration (ECS).",
+  "description": "Detect anomalies in transactions from your APM services",
```

Comment:
Nit - but we always use a period `.` at the end of these descriptions.

Reply:
got it - made the change.